README.md

```diff
@@ -216,6 +216,23 @@ You're probably thinking something like: "this is a large amount of work just to
 1.  Forces decoupling. In our example, this is illustrated by decoupling our configuration and database configuration.
 2.  After a type is configured, it can be injected anywhere with no additional effort. Simply `@inject` and it appears. We don't really illustrate that here, but you can imagine adding an arbitrary number of `RequestHandler` subclasses, all of which will automatically have a DB connection provided.
 
+
+First time setup in development mode
+------------------------------------
+
+Install `Injector` in editable mode with development dependencies:
```

Comment:
I don't want to encourage or suggest installing the package in development mode, I have bad experience with this. Instead let's suggest installing development dependencies from `requirements-dev.txt`.

Reply:
>I have bad experience with this.

Too subjective argument.

We should use development mode because
1. It's the way how Python packages are developed locally (it will provide proper linking and other things)
2. Other popular packages use it

>Instead let's suggest installing development dependencies from requirements-dev.txt

We should use the extras_require feature for a specific installation, it was developed for such cases.
it has nothing to do with development mode.
`pip install '.[dev]'`